.github/workflows/ci-sharkfuser.yml

```diff
@@ -63,73 +59,49 @@ jobs:
               -DCMAKE_LINKER_TYPE=LLD
               -DSHARKFUSER_DEBUG_BUILD=ON
               -DSHARKFUSER_CODE_COVERAGE=OFF
-            additional-packages: clang lld
-            env:
-              FUSILI_LOG_INFO: 1
-              FUSILI_LOG_FILE: stdout
           - name: Ubuntu (GCC 13, Code Coverage)
             runs-on: ubuntu-24.04
             cmake-options:
               -DCMAKE_C_COMPILER=gcc-13
               -DCMAKE_CXX_COMPILER=g++-13
               -DSHARKFUSER_DEBUG_BUILD=OFF
               -DSHARKFUSER_CODE_COVERAGE=ON
-            additional-packages: lcov
-            env:
-              FUSILI_LOG_INFO: 0
-              FUSILI_LOG_FILE: stdout
 
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+    - name: Checkout sharkfuser
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
-    - name: Forward matrix environment variables
-      run: |
-        echo "FUSILI_LOG_INFO=${{ matrix.env.FUSILI_LOG_INFO }}" >> $GITHUB_ENV
-        echo "FUSILI_LOG_FILE=${{ matrix.env.FUSILI_LOG_FILE }}" >> $GITHUB_ENV
-
-    - name: Install dependencies
-      working-directory: ${{ env.SHARKFUSER_DIR }}
-      run: |
-        sudo apt update
-        sudo apt install cmake ninja-build stow catch2 wget ${{matrix.additional-packages}}
-        python -m venv --prompt fusili .venv
-        source .venv/bin/activate
-        python -m pip install --upgrade pip
-        pip install -r ./test_requirements.txt
 
-    - name: Install prebuilt iree
-      working-directory: ${{ env.SHARKFUSER_DIR }}
-      run: |
-        wget --no-verbose ${{ env.IREE_RELEASE_URL }} -O /tmp/iree-dist.tar.xz
-        sudo mkdir -p /usr/local/stow/iree-dist
-        sudo tar -xf /tmp/iree-dist.tar.xz -C /usr/local/stow/iree-dist
-        sudo stow -d /usr/local/stow -t /usr/local iree-dist
-        rm /tmp/iree-dist.tar.xz
+    - name: Checkout docker
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        repository: sjain-stanford/docker
+        path: docker
+        ref: 57e046162842b7784d52c3a80035ca2609f1557a
```

Comment:
I'm not sure how hard this would be, but it would be nice if this Dockerfile lived in this repository and the repo wasn't a personal one. If there's an AMD OSS docker repo that would probably be ideal.